plugins/action_remove_user_org/src/removeUserOrg.ts

```diff
@@ -0,0 +1,41 @@
+import { ActionQueueStatus } from '../../../src/generated/graphql'
+import { ActionPluginInput } from '../../types'
+
+const removeUserOrg = async function ({ parameters, DBConnect }: ActionPluginInput) {
+  const { userId, orgId, user_id, organisation_id, org_id, deletePermissions = true } = parameters
+  const userOrg = {
+    userId: userId ?? user_id,
+    orgId: orgId ?? org_id ?? organisation_id,
+  }
+  try {
+    console.log(`\Removing user from organisation...`)
+    const result = await DBConnect.removeUserOrg(userOrg)
+
+    if (deletePermissions) await DBConnect.deleteUserOrgPermissions(userOrg)
```

Comment:
I didn't bother collecting the result of this one, because it's not critical, and if the first operation (the actual remove user from org) was successful then I'd consider the action a success. 

Anything else would muddy the waters if the user was actually removed.

Also -- barely any chance of this one failing if the first mutation succeeded.